source/User_Guide/Marketing_Campaigns/senders.md

```diff
@@ -22,37 +22,36 @@ Before you begin, go to your SendGrid [account settings]({{site.app_url}}/user/a
 Adding a Sender
 {% endanchor h2 %}
 
-1. Click **Create New Sender**
-1. Fill out the entire "Add Sender Identity" form. This form is so extensive so that you are CAN-SPAM compliant. The footer of your emails includes the information you provide here.
+This is to ensure that you are CAN-SPAM compliant. The footer of your emails includes the information you provide here.
```

Comment:
this seems like an incomplete thought - how about "The footer of your emails automatically contains your sender information, because CAN-SPAM regulations require this information to be in every email you send."